projects/plugins/mu-wpcom-plugin/package.json

```diff
@@ -1,7 +1,7 @@
 {
 	"private": true,
 	"name": "@automattic/jetpack-mu-wpcom-plugin",
-	"version": "1.0.11-alpha",
+	"version": "1.0.11-beta",
```

Comment:
We've been doing stable releases for this plugin, not any alpha/beta cycles. Did you select `stable` when running the changelogger for this one?

Reply:
Whoops, I used a `-b` switch here.